### PR TITLE
data: Sunkio

### PR DESCRIPTION
**Pull request by Sunkio**
## Check List (Check all the applicable boxes) 

- [ x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Thanks for reviewing my PR :)

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4203"></a>

### CODE REVIEW

**Review by dr-adada — DISMISSED** [review]
This looks great!

File: data/Sunkio.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Tanja Schmidt",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web & decentralized tech enthusiast. Software Developer keen on building web & blockchain solutions.",
+  "avatar": "https://github.com/Sunkio.png",
```
Comment (Pradumnasaraf) [inline data/Sunkio.json:6]:
```suggestion
```

**Review by Pradumnasaraf — DISMISSED** [review]
Hey,

Everything looks good. In the recent update, we don't need to add `avatar:` anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart

File: data/Sunkio.json (hunk 1 above)
Comment (Sunkio) [reply data/Sunkio.json]: Thanks for your time and your feedback! I removed the link. Hope everything's okay now. :)

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Ananya2001-an** [comment]
@Pradumnasaraf requested changes have been made by @Sunkio. You can close the review now.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good to merge. Thanks for your Contribution! 🎉 

<p align="center"></p>

**Comment by Sunkio** [comment]
Great, thank you!

**Pull request merged**